src/content/en/updates/2017/03/background_tabs.md

```diff
@@ -0,0 +1,70 @@
+# Background tabs in Chrome 57.
+
+Background tabs can have a dramatic negative effect on browser performance, especially on battery life. To mitigate this, Chrome have been placing various restrictions on background tabs for last several years. Recently there’s been a number of efforts to make further improvements to the situation, and this document gives an overview of what Chrome policy is. This document focuses on describing current policies in Chrome 57. Long-term strategy and further plans can be found in [this document](https://docs.google.com/document/d/1FCcMdUDl25RCnzfAbKOur5yVACCjS1EV4PUvlPRQ-9w/edit).
+
```

Comment:
the link to google docs takes me to an empty doc, is that intentional? It would also be good to use the /pub version instead so people aren't able to edit, comment, etc.